### PR TITLE
Ability to run project with docker-compose

### PR DESCRIPTION
**Pull request by DepkaCZ**
Hi,

I've added the ability to run this project with docker compose, which i find way more easier for everyone else that want's to try it.

### CODE REVIEW

**Comment by f3l1x** [comment]
Thanks, I've reviewed it and there're some points:

1. Rename /docker to /.docker
2. Don't upgrade nette/finder
3. Don't rename default database host
4. Create two different "how to start" manuals, first without docker and second with docker. 

Thank you very much. 

PS: Would be able to prepare same setup for nuttela-project? It would be awesome.

**Comment by f3l1x** [comment]
ping @Deepress

**Comment by Deepress** [comment]
I'll get onto that in this week :)

**Comment by f3l1x** [comment]
Please upgrade it to PHP 7.4 and latest node, thanks.

**Comment by Deepress** [comment]
Requested changes are worked in, please check.

File: .docker/nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    access_log  /var/log/nginx/access.log;
+    # Switch logging to console out to view via Docker
+    #access_log /dev/stdout;
```
Comment (f3l1x) [inline .docker/nginx/nginx.conf:18]: We should think Docker-way only these days.

File: .docker/nginx/sites/default.conf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        #fixes timeouts
+        fastcgi_read_timeout 600;
+        include fastcgi_params;
+    }
+
+    location ~ /\.ht {
+        deny all;
+    }
+
+    location /.well-known/acme-challenge/ {
```
Comment (f3l1x) [inline .docker/nginx/sites/default.conf:30]: Keep it to some proxy.

File: .docker/nginx/conf.d/default.conf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+upstream php-upstream {
+    server php:9000;
+}
```
Comment (f3l1x) [inline .docker/nginx/conf.d/default.conf:3]: New line please.

File: .docker/nginx/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,5 @@
+FROM nginx:alpine
+COPY ./ /var/www/html/
+CMD ["nginx"]
+
+EXPOSE 80 443
```
Comment (f3l1x) [inline .docker/nginx/Dockerfile:5]: New line please.

File: .docker/nginx/nginx.conf [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    #error_log /dev/stderr;
+
+    sendfile        on;
+    keepalive_timeout  65;
+
+    include /etc/nginx/conf.d/*.conf;
+    include /etc/nginx/sites-available/*.conf;
+}
```
Comment (f3l1x) [inline .docker/nginx/nginx.conf:26]: New line please.

File: .docker/nginx/sites/default.conf [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        root /var/www/letsencrypt/;
+        log_not_found off;
+    }
+}
```
Comment (f3l1x) [inline .docker/nginx/sites/default.conf:34]: New line please.

**Comment by f3l1x** [comment]
Please rebase it correctly.

image

File: .docker/nginx/nginx.conf (hunk 1 above)
Comment (DepkaCZ) [reply .docker/nginx/nginx.conf:18]: I don't understand, please explain.

File: .docker/nginx/sites/default.conf (hunk 2 above)
Comment (DepkaCZ) [reply .docker/nginx/sites/default.conf:30]: I don't understand, please explain.

**Comment by DepkaCZ** [comment]
> Please rebase it correctly.
> 
> image

What is the correct way plase ?

File: .docker/nginx/nginx.conf (hunk 1 above)
Comment (f3l1x) [reply .docker/nginx/nginx.conf:18]: Uncomment `access_log /dev/stdout;` a drop logging to file.

File: .docker/nginx/sites/default.conf (hunk 2 above)
Comment (f3l1x) [reply .docker/nginx/sites/default.conf:30]: Drop these lines and keep certificates to proxies.

**Comment by f3l1x** [comment]
> > Please rebase it correctly.
> > image
> 
> What is the correct way plase ?

Rebase it properly. You have made only 4 commits.

image

**Comment by DepkaCZ** [comment]
All  requests done, please check

**Pull request merged**